src/Our.Umbraco.Ditto/Ditto.cs

```diff
@@ -28,7 +27,7 @@ public static class Ditto
         /// <summary>
         /// The default processor cache by flags
         /// </summary>
-        public static DittoCacheBy DefaultCacheBy = DittoCacheBy.ContentId | DittoCacheBy.ContentVersion | DittoCacheBy.PropertyName | DittoCacheBy.Culture;
+        public static DittoCacheBy DefaultCacheBy = DittoCacheBy.ContentId | DittoCacheBy.UpdateDate | DittoCacheBy.PropertyName | DittoCacheBy.Culture;
```

Comment:
ContentVersion not on IPublishedContent anymore, using UpdateDate is probably equivilent?